lib/form/js/repeatable.js

```diff
@@ -4,257 +4,338 @@
 (function formRepeatable() {
   'use strict';
 
+  // Disabled one-var and func-names for better readability
```

Comment:
Due to func-names, we will have two names for a same function
Do you want me to add another name to all functions?